ts/features/fci/utils/dates.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Returns the number of days between two dates
+ */
+export const daysBetweenDate = (d1: Date, d2: Date) =>
+  Math.abs(d1.getTime() - d2.getTime()) / (1000 * 60 * 60 * 24);
```

Comment:
Is that the expected behavior? Do you really want to only check the number of days (positive), by using `Math.abs` loosing the information about the temporal timeline of d1 and d2?

Reply:
as above